.github/workflows/publish.yaml

```diff
@@ -1,27 +1,31 @@
 name: "Publish container images"
 on:
   workflow_dispatch:
-  workflow_call:  # Workflow is meant to be called from another workflow, with the image tag as input
+  workflow_call: # Workflow is meant to be called from another workflow, with the image tag as input
     inputs:
       image-tag:
-        description: 'Which tag to set for the produced docker images'
-        default: 'latest'
+        description: "Which tag to set for the produced docker images"
+        default: "latest"
         required: true
         type: string
       oauth-redirect-url:
-        description: 'Redirect url for oauth. Should be the public url to access the web app'
-        default: 'https://proxy-mercury-dev.radix.equinor.com'
+        description: "Redirect url for oauth. Should be the public url to access the web app"
+        default: "https://proxy-mercury-dev.radix.equinor.com"
         required: true
         type: string
+      release-tag:
```

Comment:
I think it would be cleaner if the workflow takes a "tags" array as input, and after building the image, runs over that array in parallel to tag and upload images.

If that's is not possible, calling the workflow twice with a single tag also helps to keep the workflows more "functional"

Reply:
Github actions only support inputs of type boolean, number or string. I could take in a comma separated string and split it up though.
If we call the workflow twice (that was what it did before) I think we'll create the image twice too. Now we create one image with two tags

Reply:
And how would you do the tagging in parallel? Jobs run in parallel, but they don't share environment, so I would have to push the image with one of the tags only to pull it down again and retag it in the next workflow.